Kitodo-DataEditor/src/test/java/org/kitodo/dataeditor/MetsKitodoWriterTest.java

```diff
@@ -123,13 +128,16 @@ public void shouldWriteMetsFileFromOldFormat()
         String loadedMetadata = metsKitodoWrapper.getDmdSecs().get(0).getKitodoType().getMetadata().get(0).getValue();
         String savedMetadata = savedMetsKitodoWrapper.getDmdSecs().get(0).getKitodoType().getMetadata().get(0).getValue();
 
-        Assert.assertEquals("The metadata of the loaded and the saved mets file are not equal", loadedMetadata,
-            savedMetadata);
-        Assert.assertEquals("The number of dmdSec elements of the loaded and the saved mets file are not equal",
-            metsKitodoWrapper.getDmdSecs().size(), savedMetsKitodoWrapper.getDmdSecs().size());
+        assertEquals(loadedMetadata,
+            savedMetadata,
+            "The metadata of the loaded and the saved mets file are not equal");
+        assertEquals(metsKitodoWrapper.getDmdSecs().size(),
+            savedMetsKitodoWrapper.getDmdSecs().size(),
+            "The number of dmdSec elements of the loaded and the saved mets file are not equal");
 
-        Assert.assertEquals("Conversation note was not inserted to mets header", 2,
-            metsKitodoWrapper.getMets().getMetsHdr().getAgent().get(0).getNote().size());
+        assertEquals(2,
+            metsKitodoWrapper.getMets().getMetsHdr().getAgent().get(0).getNote().size(),
+            "Conversation note was not inserted to mets header");
```

Comment:
```suggestion
            "Conversion note was not inserted to mets header");
```
I think this should be `Conversion`, not `Conversation` here. (I known you are not responsible for the original text, but while you are at it, would you mind fixing this little error as well?)

Reply:
Sure I can and will change it to your suggestion.